src/EPR.Payment.Facade.Common.UnitTests/Mappings/AutoMapperTests.cs

```diff
@@ -32,39 +33,43 @@ public void AutoMapper_Configuration_IsValid()
             configAction.Should().NotThrow();
         }
 
-        [TestMethod, AutoMoqData]
-        public void PaymentRequestDto_To_GovPayPaymentRequestDto_Mapping_IsValid(PaymentRequestDto paymentRequestDto)
+        [TestMethod]
+        public void PaymentRequestDto_To_GovPayRequestDto_Mapping_IsValid()
         {
-            var govPayPaymentRequestDto = _mapper.Map<GovPayRequestDto>(paymentRequestDto);
+            var fixture = new Fixture();
+            var paymentRequestDto = fixture.Create<PaymentRequestDto>();
+            var govPayRequestDto = _mapper.Map<GovPayRequestDto>(paymentRequestDto);
 
             using (new FluentAssertions.Execution.AssertionScope())
             {
                 if (paymentRequestDto.Amount.HasValue)
                 {
-                    govPayPaymentRequestDto.Amount.Should().Be(paymentRequestDto.Amount.Value);
+                    govPayRequestDto.Amount.Should().Be(paymentRequestDto.Amount.Value);
                 }
 
-                govPayPaymentRequestDto.Reference.Should().Be(paymentRequestDto.Reference);
+                govPayRequestDto.Reference.Should().Be(paymentRequestDto.Reference);
 
                 if (paymentRequestDto.OrganisationId.HasValue)
                 {
-                    govPayPaymentRequestDto.OrganisationId.Should().Be(paymentRequestDto.OrganisationId.Value);
+                    govPayRequestDto.OrganisationId.Should().Be(paymentRequestDto.OrganisationId.Value);
                 }
 
                 if (paymentRequestDto.UserId.HasValue)
                 {
-                    govPayPaymentRequestDto.UserId.Should().Be(paymentRequestDto.UserId.Value);
+                    govPayRequestDto.UserId.Should().Be(paymentRequestDto.UserId.Value);
                 }
 
-                govPayPaymentRequestDto.Regulator.Should().Be(paymentRequestDto.Regulator);
-                govPayPaymentRequestDto.return_url.Should().BeNull(); // Ignored in mapping
-                govPayPaymentRequestDto.Description.Should().BeNull(); // Ignored in mapping
+                govPayRequestDto.Regulator.Should().Be(paymentRequestDto.Regulator);
+                govPayRequestDto.return_url.Should().BeNull(); // Ignored in mapping
+                govPayRequestDto.Description.Should().BeNull(); // Ignored in mapping
             }
         }
 
-        [TestMethod, AutoMoqData]
-        public void PaymentRequestDto_To_InsertPaymentRequestDto_Mapping_IsValid(PaymentRequestDto paymentRequestDto)
+        [TestMethod]
+        public void PaymentRequestDto_To_InsertPaymentRequestDto_Mapping_IsValid()
         {
```

Comment:
Use AutoMoqdata to inject the dependencies wherever possible. See the comment above

Reply:
Sure will do.